test/test-network/systemd-networkd-tests.py

```diff
@@ -3277,6 +3277,106 @@ def test_ipv6_route_prefix(self):
         print(output)
         self.assertRegex(output, '2001:db8:0:1::/64 proto ra')
 
+class NetworkdMTUTests(unittest.TestCase, Utilities):
+    links = ['dummy98']
+
+    units = [
+        '12-dummy.netdev',
+        '12-dummy-mtu.netdev',
+        '12-dummy-mtu.link',
+        '12-dummy.network',
+        ]
+
+    def setUp(self):
+        remove_links(self.links)
+        stop_networkd(show_logs=False)
+
+    def tearDown(self):
+        remove_log_file()
+        remove_links(self.links)
+        remove_unit_from_networkd_path(self.units)
+        stop_networkd(show_logs=True)
+
+    def check_mtu(self, mtu, ipv6_mtu=None, reset=True):
+        if not ipv6_mtu:
+            ipv6_mtu = mtu
+
+        # test normal start
+        start_networkd()
+        self.wait_online(['dummy98:routable'])
+        time.sleep(1)
```

Comment:
Is it necessary? `systemd-networkd-tests.py` is already very slow. We should not add unnecessary `sleep()`.

Reply:
I agree!

I was getting failures at this point due to the ipv6 mtu not being set yet, however it may be when i still had this as 'degraded' instead of routable; let me run some tests to see if the sleep is still needed (and if so, I'll try to figure out why, as it shouldn't be needed).

Reply:
The problem was the new tests were restarting networkd too many times, too fast; I added a commit to set StartLimitIntervalSec=0 to prevent the quick multiple restarts from failing.

This allows removing this and the other added sleep()s